test/e2e/scripts/prow_run_smoke_test.sh

```
@@ -172,12 +172,30 @@ setup_vars_for_tests() {
 
     # Export INSECURE_HTTP for smoke.sh (it handles MAAS_API_BASE_URL detection)
+    # HTTPS is the default for MaaS.
+    # HTTP is used only when INSECURE_HTTP=true (opt-out mode).
     # This aligns with deploy-openshift.sh which also respects INSECURE_HTTP
     export INSECURE_HTTP
     if [ "$INSECURE_HTTP" = "true" ]; then
         echo "⚠️  INSECURE_HTTP=true - will use HTTP for tests"
     fi
-    
+       
+    export CLUSTER_DOMAIN="$(oc get ingresses.config.openshift.io cluster -o jsonpath='{.spec.domain}')"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)